packages/ui/src/DataTable/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from 'tamagui';
+
+import {
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
+import { createColumns } from './columnsParser';
+import { DataTable } from './Table';
+ 
+
+function App({ data = [{}], resize = false }) {
+
+  const columns = createColumns(data);
+  const table = useReactTable({
+    data,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+    enableColumnResizing:true,
+    columnResizeMode:  'onChange'
+  });
+
+  return (
+    <div className="p-2">
```

Comment:
does this need to be a div ? we should probably use a tamagui stack or box here so it works on native